support-frontend/assets/pages/paper-subscription-checkout/components/paperCheckoutForm.tsx

```diff
@@ -567,7 +567,7 @@ function PaperCheckoutForm(props: PropTypes) {
 						/>
 					) : null}
 					<GeneralErrorMessage
-						errorReason={props.submissionError}
+						errorReason={props.submissionError ?? undefined}
```

Comment:
Is this needed because `props.submissionsError` could be `null` but `GeneralErrorMessage` wants it to be something or `undefined`?

Reply:
It's because I removed the `Option` type from `GeneralErrorMessage`, an unnecessary type in TS, which could have had the value as `T | undefined | null`. 

It's now optional - as in `val?: string` - so this was coalescing the `null` into `undefined`.